.github/workflows/run_tests.yml

```diff
@@ -57,7 +57,7 @@ jobs:
         os: [macos-12, ubuntu-20.04] # Operating systems
         compiler: [8] # GNU compiler version
         rai: [1.2.7] # Redis AI versions
-        py_v: ['3.8', '3.9', '3.10', '3.11'] # Python versions
+        py_v: ['3.9', '3.10', '3.11'] # Python versions
```

Comment:
If we make this change, should it be [mirrored](https://github.com/CrayLabs/SmartRedis/blob/develop/.github/workflows/run_tests.yml#L67) in SmartRedis?

Reply:
ditto in `release` [action](https://github.com/al-rigazzi/SmartSim/blob/protodrg/.github/workflows/release.yml#L101)

Reply:
ditto in [docs](https://github.com/CrayLabs/SmartSim/blob/develop/doc/installation_instructions/basic.rst?plain=1#L23)